### DISCUSSION TITLE
Show user stats for all repo contributed

### DISCUSSION THREAD

**Discussion by hussamkhatib** [lowlighter/metrics#654] [Ideas]
Feature request : It would be nice if we can  get all the list of repo's a that  user has contributed  to .

**Comment by lowlighter** [comment]
Thanks for your suggestion!
I'll dig into it 👍

**Comment by jayvdb** [comment]
Is this about a list of repos?  The [notable](https://github.com/lowlighter/metrics/tree/master/source/plugins/notable) plugin sort of does this, I thought, if `plugin_notable_from` is set to `all`, and `plugin_notable_repositories` is enabled.

Or a single stat "Contributed to x repositories" ?  I wonder if this is one of the existing badges in the [achievements](https://github.com/lowlighter/metrics/tree/master/source/plugins/achievements) plugin?

**Reply by lowlighter** [reply]
Yeah indeed, the notable plugin sort of does this, except it doesn't include your own repositories (only those from others).
But we could add an option for it

The single stat is already present too, it's in the base header just below the one-week calendar:
![image](https://user-images.githubusercontent.com/22963968/142077400-019079d5-5c4a-4fc8-9589-0bbc60eca6ca.png)

**Reply by jayvdb** [reply]
Cross-linking a ramble from me at https://github.com/lowlighter/metrics/discussions/1090#discussioncomment-3188243 , regarding having a threshold for "notable self-created repositories" in order to avoid all the junky ones, and using a default of 16 stars in line with the GitHub "Starstruck" achievement.
